cms/jinja2/templates/pages/methodology_page.html

```diff
@@ -87,14 +87,31 @@
     <div class="ons-grid ons-grid-flex-gap ons-grid-flex-gap--32 ons-js-toc-container">
         <div class="ons-grid__col ons-grid__col--sticky@m ons-col-4@m">
             {% with toc_title=_("Contents"), toc_aria_label=_("Sections in this page") %}
+
+                {% if related_publications %}
+                    {% set related_publications_title = _("Related publications") %}
+                    {% set related_publications_list = related_publications %}
+                {% endif %}
+
                 {# fmt:off #}
                 {{-
                     onsTableOfContents({
                         "title": toc_title,
                         "ariaLabel": toc_aria_label,
-                        "itemsList": table_of_contents
+                        "lists": [
+                            {
+                                "itemsList": table_of_contents
```

Comment:
q: https://service-manual.ons.gov.uk/design-system/components/table-of-contents#example.example-table-of-contents-sticky.nunjucks still uses `itemsList`. Is the plane to deprecate this, hence moving to `lists` or is this the mix of the TOC items + related links

Reply:
Good question! Lists allows us to add multiple itemsList if needed but as we only have one here its unnecessary so I've removed for simplicity 👍  I think I'd used the structure from an example in the design system which only had one as well